### DISCUSSION TITLE
Feature to Split .hex & .bin files (for writing to EPROM & FLASH chips)

### DISCUSSION THREAD

**Discussion by JDW1** [HexFiend/HexFiend#401] [Ideas]
There is an app called [SRecord](https://srecord.sourceforge.net)  that runs on Windows and Linux but not MacOS, and it can split .bin or .hex files into a user specified number of parts.  It would be nice if a MacOS app could do that, which prompted me to think about HexFiend.  Here is [one example](https://tinkerdifferent.com/threads/smc-rom-simm-prototype.2198/) of why you would want to do that — to program file contents onto 4 PLCC flash memory chips for use on a ROM SIMM used in vintage Macs.

For that reason, I wanted to add File Splitting as an Idea for a very nice enhancement to HexFiend.

Thanks.

**Comment by kainjow** [comment]
Sounds similar to this request? https://github.com/HexFiend/HexFiend/issues/259

**Comment by kainjow** [comment]
It's on Homebrew, so you can install it https://formulae.brew.sh/formula/srecord
